src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/CovariantEqualsCheck.java

```diff
@@ -50,7 +50,7 @@
  * Finding Bugs is Easy, chapter '4.5 Bad Covariant Definition of Equals (Eq)'</a>:
  * </p>
  * <p>
- * Java classes may override the {@code equals(Object)} method to define
+ * Java classes and records may override the {@code equals(Object)} method to define
```

Comment:
I think it is a good idea to update code examples with record example

Reply:
Done.